core/server/api/v2/utils/serializers/input/tags.js

```diff
@@ -0,0 +1,9 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:input:tags');
+const url = require('./utils/url');
+
+module.exports = {
+    edit(apiConfig, frame) {
```

Comment:
And while at it, maybe let's add `feature_image` parameter to our basic [functional suite](https://github.com/TryGhost/Ghost/blob/master/core/test/functional/api/v2/admin/tags_spec.js#L145) :wink: